using-ai-extensions/7-aws-unit-test.js

```diff
@@ -0,0 +1,20 @@
+function mood(isHappy) {
```

Comment:
Hi folks, 
Thanks for raising this PR - can you come up with useful well-named utility functions instead of functions like `mood` or `greaterThan10`. If you're making use of Co-pilot then you can feel free to be more adventurous.